install.sh

```diff
@@ -79,6 +79,7 @@ ln -s $TRAVIS_BUILD_DIR $HOME/arduino_ide/libraries/Adafruit_Test_Library
 
 # add the arduino CLI to our PATH
 export PATH="$HOME/arduino_ide:$PATH"
+export PATH="$HOME/.local/bin/:$PATH"
```

Comment:
I tried to add that so github would exec nrfutils but didn't worked, I had to sudo install them